.babelrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I added this file back in so jest tests would succeed. I had to take out one extra line that rollup was complaining about: https://github.com/mobxjs/mst-gql/pull/288/commits/72495b3aedd08233aa55e09d2c2e2ce1c9588edd

I tried to use [rollup-jest](https://github.com/ambar/rollup-jest) but tests hang and never succeed.